backend/main.py

```diff
@@ -10,6 +13,10 @@
 from src.core.auth_middleware import RequireAuth, OptionalAuth
 from src.utils.reminder_integration import lifespan
 
+# Set timezone to IST (India Standard Time) globally
+os.environ['TZ'] = 'Asia/Kolkata'
+time.tzset()
```

Comment:
This file has error on Windows because `time.tzset()` only works on Unix. Since deployment env supports `tzset`, guarding it should be enough:

```suggestion
if hasattr(time, 'tzset'):
    time.tzset()
```